### PR TITLE
fix simulation doc issue

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: doc/simulation/local-policies/README.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ spec:
         name: storage-secret
     type: largeScale
 ```
+Note: You may need to recreate the multicluster-global-hub-manager deployment after the `MGH` instance updated.
+```bash
+oc delete deploy/multicluster-global-hub-manager -n open-cluster-management
```
Comment (clyang82) [inline doc/simulation/local-policies/README.md:40]: Can we just delete pod?

Comment (morvencao) [reply doc/simulation/local-policies/README.md:40]: Removing operator should alsow work, currently the operator won't watch annotation change in MGH instance.

Comment (ldpliu) [reply doc/simulation/local-policies/README.md:40]: I am not sure if just remove pod works, as it looks the the info need to add to the deployment flags.

Comment (ldpliu) [reply doc/simulation/local-policies/README.md]: Changed

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**